src/test/java/seedu/address/testutil/TypicalPatients.java

```diff
@@ -26,11 +26,11 @@ public class TypicalPatients {
     public static final Patient ALICE = new PatientBuilder().withName("Alice Pauline")
             .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
             .withPhone("94351253")
-            .withTags("friends").build();
-    public static final Patient BENSON = new PatientBuilder().withName("Benson Meier")
-            .withAddress("311, Clementi Ave 2, #02-25")
-            .withEmail("johnd@example.com").withPhone("98765432")
-            .withTags("owesMoney", "friends").build();
+            .withTags("friends")
+            .withBirthday("12/03/2001")
+            .withGender("FEMALE")
+            .withIcNumber("T1234567N").build();
```

Comment:
I like how you have added the new Patient fields for ALICE typical Patient Object